core/templates/pages/blog-dashboard-page/blog-card/blog-card.component.ts

```diff
@@ -0,0 +1,67 @@
+// Copyright 2021 The Oppia Authors. All Rights Reserved.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS-IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+/**
+ * @fileoverview Component for a blog card.
+ */
+
+import { Component, Input, OnInit } from '@angular/core';
+import { downgradeComponent } from '@angular/upgrade/static';
+import { AppConstants } from 'app.constants';
+import { BlogPostSummary } from 'domain/blog/blog-post-summary.model';
+import { AssetsBackendApiService } from 'services/assets-backend-api.service';
+import { UrlInterpolationService } from 'domain/utilities/url-interpolation.service';
+import dayjs from 'dayjs';
+@Component({
+  selector: 'oppia-blog-card',
+  templateUrl: './blog-card.component.html'
+})
+export class BlogCardComponent implements OnInit {
+  @Input() blogPostSummary: BlogPostSummary;
+  @Input() authorProfilePicDataUrl: string;
```

Comment:
```suggestion
  @Input() blogPostSummary!: BlogPostSummary;
  @Input() authorProfilePicDataUrl!: string;
```
Please try to use not null assertion for @Input variables

Reply:
Done. Thanks.